src/frontends/onnx/frontend/src/op/reduce.cpp

```diff
@@ -93,24 +93,28 @@ std::shared_ptr<ov::Node> get_reduction_axes_from_attr(const Node& node) {
 template <typename OpType>
 std::shared_ptr<ov::Node> make_ng_reduction_op(const Node& node,
                                                const ov::Output<ov::Node>& ng_input,
-                                               bool axes_as_attr = true) {
+                                               bool axes_as_attr = true,
+                                               const std::pair<int64_t, int64_t>& axes_range = {
+                                               -std::numeric_limits<int64_t>::max(),
+                                               std::numeric_limits<int64_t>::max()}) {
     const std::int64_t keepdims = node.get_attribute_value<std::int64_t>("keepdims", 1);
 
     const auto reduction_axes = axes_as_attr ? get_reduction_axes_from_attr(node) : get_reduction_axes_from_input(node);
     if (reduction_axes != nullptr) {
+        // Check that axes are within the specified range
```

Comment:
I think you may not need such overcomplication after rebasing to a fresh master.

Reply:
> I think you may not need such overcomplication after rebasing to a fresh master.

Ok, thanks for the advice!